bbl/i18n/de/BambuStudio_de.po

```diff
@@ -988,125 +1000,128 @@ msgid ""
 "Tip:If you want to place svg file on another part surface,\n"
 "you should select part first, and then drag svg file to the part surface."
 msgstr ""
+"Tipp: Wenn Sie die svg-Datei auf der Fläche eines Teils platzieren möchten,\n"
+"wählen Sie zuerst das Teil und ziehen Sie dann die svg-Datei auf die "
+"Oberfläche des Teils."
 
 #. TRN - Preview of filename after clear local filepath.
 msgid "Unknown filename"
-msgstr ""
+msgstr "Unbekannter Dateiname"
 
 #, boost-format
 msgid "SVG file path is \"%1%\""
 msgstr ""
 
 msgid "Reload SVG file from disk."
-msgstr ""
+msgstr "SVG-Datei neu laden."
 
 msgid "Change file"
-msgstr ""
+msgstr "Datei ändern"
 
 msgid "Change to another .svg file"
-msgstr ""
+msgstr "Eine andere SVG-Datei laden"
 
 #. TRN: An menu option to convert the SVG into an unmodifiable model part.
 msgid "Bake to model"
-msgstr ""
+msgstr "In das Modell einbrennen"
```

Comment:
Better translation would be: "mit dem Model zusammenführen"

Reply:
Thank you for the suggestion, I will adapt this.